README.md

```diff
@@ -19,6 +47,54 @@ For formatting code automatically please use `poetry run autopep8 --in-place --r
+  - [Checks](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/base.py#L14) if we should process the library according to its [requested frequency](https://searchfox.org/mozilla-central/rev/83e67336083df9f9a3d1e0c33f2ba19703d57161/python/mozbuild/mozbuild/vendor/moz_yaml.py#392). If so it continues...
+  - [Compares](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L51-L54) the current upstream revision with the current [in-tree revision](https://searchfox.org/mozilla-central/rev/83e67336083df9f9a3d1e0c33f2ba19703d57161/media/libdav1d/moz.yaml#27). If there is an update it continues...
+  - It [checks if it's seen this (new) revision before](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L57).  We will assume it has not, so it...
+  - ... [files a bugzilla bug](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L154)...
+  - ... [vendors](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/mach_vendor.py#L42-L43) the latest version from upstream by calling `./mach vendor manifest path/to/moz.yaml` ...
+  - ... [checks the result](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L159-L171) ...
+  - ... [commits](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/hg.py#L14-L19) the change locally...
+  - ... [submits it to try](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/apis/taskcluster.py#L51) ...
+  - ... [comments on the bug](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L179) ...
+  - ... and (finally) [adds it to the Updatebot database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L181).
+- That database lives in Google CloudSQL.  There is a dev and prod database, as well as [dev and prod credentials](https://searchfox.org/mozilla-central/rev/83e67336083df9f9a3d1e0c33f2ba19703d57161/taskcluster/docker/updatebot/run.py#79-84) for those databases, bugzilla, try server, phabricator, sentry, and sql-proxy (which is used to connect to the database).  You can find them in [grants.yml](https://hg.mozilla.org/ci/ci-configuration/file/tip/grants.yml#l644) searching for 'updatebot'.  The dev creds are granted to holly, which is our reserved development instance because we can't test it o try.  The prod creds are only available on mozilla-central.
```

Comment:
This line might fit better in the architecture section below.
`try server` -> `treeherder`
`creds` -> `credentials`
`we can't test it o try.` -> `Updatebot can't be tested on try.`
`creds` -> `credentials`
`available on mozilla-central` -> `available for mozilla-central`

Reply:
the creds are actually for the try server, not tree herder, so leaving that.